docs/design/mockups/phase-4-homepage/round-8b-youthblock-imagery-comparisons.html

```
@@ -0,0 +1,485 @@
+      </section>
+
+      <!-- N.3 — Pattern of small JerseyShirts -->
+      <section class="candidate" id="N3">
+        <header>
+          <div>
+            <div class="key">N.3</div>
+            <div class="name">Decorative pattern · small jerseys cluster</div>
+            <div class="desc">
+              Several small JerseyShirts arranged on the right side as a low-opacity decorative
+              cluster. They're atmosphere, not subject. JerseyShirt vocabulary stays alive but at
+              the right scale (multiple small > one big). No real photo needed.
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)